build/defines.mak

```diff
@@ -69,4 +69,10 @@ DEFAULT_RST_FILES_TO_GENERATE := \
                      attributes.rst \
                      functions.rst \
 
+# Files for tracking parts of the build
```

Comment:
I think the names of these variables / files should make it obvious that they are related and that they are used for the same purpose.

How about _DONE / done ?

Reply:
GENERATED_FILES_COPIED would become GENERATED_FILES_COPY_DONE

Reply:
_DONE

Reply:
we should rename the file name as well, right?

Reply:
I am working on another change to do most of this through tox directly with a single call, so these will be going away.